### ISSUE TITLE
Useful Documentation is Hard to View

### ISSUE THREAD

**Issue by cspalding** [launchdarkly/api-client-ruby#3]
## Description of the issue
Hey there,
I'm working on integration this API Client into a small project of mine. There's a bunch of useful documentation in the README, but it's hidden by pages of text that are really hard to parse. Could there be a formatting error in the README? Would it possible to fix the formatting issue so that the docs are easier to access without so much scrolling? Thanks in advance for any help you can provide!

## Expected
When viewing the README, text is intuitively formatted and easy to read so that developers have quick access to information about integrating the LD API Ruby Client into their projects.

## Actual
Useful information about using the client is hidden by lots of hard-to-read text

## Screenshots
<img width="934" alt="Screen Shot 2021-09-29 at 12 01 31 PM" src="https://user-images.githubusercontent.com/5385144/135332671-8ceb9027-f46f-4b5e-97a7-02ff94ec4ab3.png">
<img width="983" alt="Screen Shot 2021-09-29 at 12 01 53 PM" src="https://user-images.githubusercontent.com/5385144/135332678-4aec9cd5-3d8f-4b04-a581-500a69f6a67b.png">

**Comment by carmenquan** [comment]
Hi @cspalding - thanks for bringing this to our attention. Our team will take a look at this, as a workaround, you can also find our api documentation at https://apidocs.launchdarkly.com/.

**Comment by carmenquan** [comment]
This issue has been addressed with our latest push changes. Thanks again for opening an issue for this @cspalding. Feel free to reach out or create an issue if there's anything else we can help with.

**Comment by cspalding** [comment]
@carmenquan thanks so much for addressing the issue 🎉! The README looks great 💯
